src/components/Nav.js

```diff
@@ -1,23 +1,28 @@
 import React from 'react';
 import { NavLink } from 'react-router-dom';
-import '../styles/Nav.css';
+import { ReactComponent as ListButton } from '../img/list-button.svg';
+import { ReactComponent as AddItemButton } from '../img/add-item-button.svg';
 
 const Nav = () => {
   return (
-    <nav>
-      <ul className="flexed-list">
-        <li className="list">
-          <NavLink to="/list" activeClassName="active">
-            List
-          </NavLink>
-        </li>
-        <li className="list">
-          <NavLink to="/addItem" activeClassName="active">
-            Add Item
-          </NavLink>
-        </li>
-      </ul>
-    </nav>
+    <div>
+      <div className="md:hidden">
+        <nav className="h-auto fixed bottom-0 right-0 left-0">
+          <ul className="flex justify-center content-center fixed bottom-0 right-0 left-0 m-auto bg-green-300 rounded-t-3xl shadow-top">
+            <li className="list-none mr-8">
+              <NavLink to="/list" activeClassName="font-bold">
+                <ListButton />
```

Comment:
I would suggest going with a real button here like:
```
<button class="rounded-full h-20 w-20 flex items-center justify-center bg-green-600 text-white shadow-xl">
<TaskIcon />
</button>   
```
and only import the icon as an SVG from https://fontawesome.com/icons/tasks?style=solid and https://fontawesome.com/icons/plus?style=solid and in the `svg` file you can set the width="40px" (e.g.)

So we can still style the button (changing the bg-color of the button). 

If there is no time for this anymore, no problem, I already set it all up and can do it in the next branch.